### DISCUSSION TITLE
Is it still not recommended to use ts-node in production NOW(2022)

### DISCUSSION THREAD

**Discussion by raphaelsoul** [TypeStrong/ts-node#1870] [Q&A] [answered]
as mentioned.

Or can we use transpile-only mode in production?

**Comment by cspotcode** [comment, marked as answer]
You are welcome to do whatever you feel is best for your project.  It is open-source, so keep in mind the "provided as-is" clause in the MIT license: https://github.com/TypeStrong/ts-node/blob/main/LICENSE

I know of people who have been using ts-node successfully in production for years and enjoy it.  If you feel it makes sense for your project, then go for it.
